.github/workflows/build-intermediary-image.yml

```
@@ -0,0 +1,116 @@
+name: Build and Push Intermediary Docker Image
+
+on:
+  push:
+  workflow_dispatch:
+
+permissions:
+  contents: write
```

```suggestion
  contents: read
```

why we need write access here?

---

Fixed